File: 1.Arrays/C++/ZeroToEnd.cpp
#include<iostream>

int main(){
    int arr[5] = {20,0,60,0,8};
    int count = 0;
    
    for (int i = 0; i < 5; i++)
    {
        if (arr[i] != 0)
        {
            arr[count++] = arr[i];
        }
    }

    while (count < 5){
        arr[count++] = 0;
    }    
    
    printf("Now array elements are: ");
    for (int i = 0; i < 5; i++) {
        printf("%d ", arr[i]);
    }
    return 0;
}

File: 1.Arrays/Kotlin/ZeroToEnd.kt
fun main() {
    var arr = arrayOf(20, 0, 60, 0, 8)
    var count = 0

    for (i in 0..4) {
        if (arr[i] != 0) {
            arr[count++] = arr[i]
        }
    }

    while (count < 5) {
        arr[count++] = 0
    }

    print("Now array elements are: ")
    println()
    for (i in 0..4) {
        print("${arr[i]} ")
    }
    println()
}
